src/lang/en.json

```diff
@@ -0,0 +1,391 @@
+{
+  "blank": "",
+  "nav": {
+    "tab": {
+      "about": "About",
+      "header": "Header",
+      "mime": "MIME types",
+      "security": "Security",
+      "tags": "Tags",
+      "paths": "Paths",
+      "definitions": "Definitions"
+    },
+    "actions": {
+      "delete": "Delete data",
+      "download": {
+        "json": "JSON",
+        "yaml": "YAML"
+      }
+    }
+  },
+  "error": {
+    "title": "Uh oh!",
+    "body": "It seems like something is broken.\nSometimes these issues are caused by an invalid OpenAPI document in the cache. If this is the case, <a href=\"javascript:void(0)\" click.delegate=\"delete()\">deleting the data</a> should fix it.",
+    "data-title": "Exception data"
+  },
+  "footer-powered-by": "Powered by",
+  "form": {
+    "externalDocs": {
+      "label": "External documentation",
+      "description": {
+        "label": "Description"
+      },
+      "url": {
+        "label": "URL"
+      }
+    },
+    "paths": {
+      "label": "Paths",
+      "newItemText": "New Path",
+      "item": {
+        "label": "Path #$index: ${#:key}",
```

Comment:
Making direct links (e.g. `{{path}}` -> `/the/path`) would break instant updating, so maybe `{{path}}` -> `${#:key}` would be better.